.devcontainer/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
+
+WORKDIR /workspaces
+
+# Default ENV
+ENV LANG C.UTF-8
+ENV DEBIAN_FRONTEND noninteractive
+
+# Set shell
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Install docker
+# https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        apt-transport-https \
+        ca-certificates \
+        curl \
+        dbus \
+        software-properties-common \
+        gpg-agent \
+        git \
+        sudo \
+    && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add - \
+    && add-apt-repository "deb https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
+    && apt-get update && apt-get install -y --no-install-recommends \
+        docker-ce \
+        docker-ce-cli \
+        containerd.io
+# This is a development container.  Don't bother to clean up apt cache, this way we have it handy later
+
+COPY .devcontainer/start_hassio.sh /usr/local/bin/start_hassio.sh
```

Comment:
We don't need copy this, let put the script into `.devcontainer`

Reply:
Agreed that this makes more sense in the generic version of the devcontainer (where users might want to change paths), but I guess in this case it can work.  I guess some pedantic part of me objects to putting a "binary" (even if it is a script) in such an oddball place...

Curious as to motives for the pednatic-ness on your end that wants to avoid the COPY, though.  Would be happy if you shared your motive.